src/Wrapper.js

```diff
@@ -7,13 +8,45 @@ import App404 from './components/App404/App404'
 import Home from './components/Home/Home'
 import MenuPanel from './components/MenuPanel/MenuPanel'
 import SignerPanel from './components/SignerPanel/SignerPanel'
+import DeprecatedBanner from './components/DeprecatedBanner/DeprecatedBanner'
 import { getAppPath } from './routing'
 import { staticApps } from './static-apps'
 import { addressesEqual } from './web3-utils'
 import { noop } from './utils'
-import { APPS_STATUS_LOADING } from './symbols'
+import {
+  APPS_STATUS_ERROR,
+  APPS_STATUS_READY,
+  APPS_STATUS_LOADING,
+} from './symbols'
 
 class Wrapper extends React.Component {
+  static propTypes = {
+    account: PropTypes.string.isRequired,
+    apps: PropTypes.array.isRequired,
+    appsStatus: PropTypes.oneOf([
+      APPS_STATUS_ERROR,
+      APPS_STATUS_READY,
+      APPS_STATUS_LOADING,
+    ]).isRequired,
+    banner: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.shape({
```

Comment:
Hmm, should this be an `instanceOf()` check instead? The react docs have [an example of this (one of many types)](https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes).

Reply:
That's where I started but kept getting a warning, then found [this](https://github.com/facebook/react/issues/2979).